### PR TITLE
#CU-2nhgxne setting up pallet config for #BYOG

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
Signed-off-by: Dzmitry Lahoda <dzmitry@lahoda.pro>

## Description

- BYOG is enabled
- fixed simulator tests local runs 
- fixed running my tests on remote (so only main remote is published as devcontainer)

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2nhgxne [BYO Gas]: XCM transfers- gas tokens of destination parachain #runtime #pallet](https://app.clickup.com/t/2nhgxne)

**Comment by dzmitry-lahoda** [comment]
oh, we have simnode running as part of all tests now. need to fix it.

**Review by KaiserKarel — APPROVED** [review]

**Comment by dzmitry-lahoda** [comment]
@Mergifyio rebase

File: docs/byog.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ Such mechanisms are built into the runtime on Picasso and [other Dotsama paracha
...
 ## Current state
 
-Any cross-chain (XCMP) transactions can choose what assets to use to pay for the transaction.
+Any cross-chain XCM transactions can choose what assets to use to pay for the transaction.
 
-Direct transaction can add header to pick up `payment` asset or accounts can configure default [payment asset](https://github.com/paritytech/substrate/discussions/12055).
+Direct transaction can add header to pick up `payment` asset or an account can configure its' default [payment asset](https://github.com/paritytech/substrate/discussions/12055).
 
-`Payment` assets are configurable and configuration is shared.
+`Payment` assets are configurable and configuration is shared withith native and XCM transaction.
```
Comment (PoisonPhang) [inline docs/byog.md:17]:
```suggestion
`Payment` assets are configurable and configuration is shared within native and XCM transactions.
```

**Review by PoisonPhang — APPROVED** [review]

**Comment by dzmitry-lahoda** [comment]
@Mergifyio rebase

**Comment by dzmitry-lahoda** [comment]
@Mergifyio merge

**Comment by dzmitry-lahoda** [comment]
@Mergifyio refresh

**Comment by dzmitry-lahoda** [comment]
mergify erased all signatures on rebase it did. hope it will work so.

**Review by PoisonPhang — APPROVED** [review]

**Comment by dzmitry-lahoda** [comment]
@Megifyio refresh

**Review by PoisonPhang — APPROVED** [review]

**Pull request merged**